crates/nu-derive-value/src/from.rs

```diff
@@ -590,6 +589,12 @@ fn parse_value_via_fields(
                                 })?,
                         )?
                     },
+                    (false, true) => quote! {
+                        #ident: record
+                            .remove(#ident_s)
+                            .and_then(|v| <#ty as nu_protocol::FromValue>::from_value(v).ok())
```

Comment:
I don't like that this impl would would silently fallback to the default. If that field exists but cannot be converted, it should throw an error.